build.gradle.kts

```diff
@@ -147,7 +161,22 @@ val targets = listOf(
     BuildTarget(image = "linux-x86", classifier = "x86_32"),
     BuildTarget(image = "linux-arm64", classifier = "aarch64"),
     BuildTarget(image = "windows-static-x64", classifier = "windows-x86_64"),
-    BuildTarget(image = "windows-static-x86", classifier = "windows-x86_32"),
+    // BuildTarget(image = "windows-static-x86", classifier = "windows-x86_32"),
+    // BuildTarget(
+    //     image = "android-arm", 
```

Comment:
I had to disable these two targets for now, because github actions doesn't allocate enough disk space for all the images. 
A possible workaround could be to remove the image after every build task, not sure how to do it with your addon @pschichtel